Modules/Example/etc/analysisDerived.json

```diff
@@ -0,0 +1,55 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "ccdb-test.cern.ch:8080"
+      }
+    },
+    "tasks": {
+      "AnalysisQcTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::example::AnalysisTask",
+        "moduleName": "QcExample",
+        "detectorName": "TST",
+        "cycleDurationSeconds": "10",
```

Comment:
Naive question: does it make sense for something running asynchronously offline to have a cycle in seconds rather than number of events ?

Reply:
in principle yes, but i would wait for any particular requirements about this.